iroha-cpp/batch-example.cpp

```diff
@@ -21,32 +21,34 @@ const auto response_handler_log_manager = log_manager->getChild("ResponseHandler
 const auto pb_qry_factory_log = log_manager->getChild("PbQueryFactory")->getLogger();
 
 
-int verifyPath(const fs::path& path, const logger::LoggerPtr& logger)
+void verifyPath(const fs::path& path, const logger::LoggerPtr& logger)
 {
     if (not fs::exists(path))
     {
-        logger->error("Path {} not found.", path.string());
-        return EXIT_FAILURE;
+        const auto error_message = "Path {} not found." + path.string();
+        logger->error(error_message);
+        throw error_message;
     }
-    return EXIT_SUCCESS;
 }
 
 
-int verifyKepair(const iroha::expected::Result<shared_model::crypto::Keypair, std::string>& keypair,
-                 const logger::LoggerPtr& logger,
-                 const fs::path& path,
-                 const std::string& account_name)
+void verifyKepair(const iroha::expected::Result<shared_model::crypto::Keypair, std::string>& keypair,
+                  const logger::LoggerPtr& logger,
+                  const fs::path& path,
+                  const std::string& account_name)
 {
     if (auto e = iroha::expected::resultToOptionalError(keypair))
     {
-        logger->error("Keypair error: {}.\n"
-                      "Keypair path: '{}', name: {}.\n",
-                      e.value(),
-                      path.string(),
-                      account_name);
-        return EXIT_FAILURE;
+        const auto error_message = "Keypair error: "
```

Comment:
According to:
https://github.com/hyperledger/iroha/blob/main/vcpkg/VCPKG_DEPS_LIST You can use: https://github.com/fmtlib/fmt instead of string concatenation

Reply:
Thanks.